kernel/kexec.c

```diff
@@ -1,7 +1,9 @@
 /*
  * fiwix/kernel/kexec.c
  *
- * Copyright 2023, Jordi Sanfeliu. All rights reserved.
```

Comment:
What is the purpose to change from 2023 to 2024 this copyright?

Reply:
I was just updating all copyrights in this file to the current year but I can revert that back to 2023 if you prefer. Please let me know.

Reply:
Yes, I think we must keep, at least, the original year as this will help to know better when this file was created.

Reply:
OK I will change that back to 2023.

Reply:
Thanks.

I still have pending to make some adjustments in the code, related to the coding style.